Kinvey.Tests/UserIntegrationTests.cs

```diff
@@ -314,6 +314,7 @@ await Assert.ThrowsExceptionAsync<KinveyException>(async delegate ()
         // MIC LOGIN TESTS
         //
         [TestMethod]
+        [Obsolete("This test has been deprecated.")]
         public void TestMIC_LoginWithMIC_NormalFlow()
```

Comment:
We are not deprecating the normal MIC flow (Authorization Grant Flow), why is this test method being deprecated?